client/my-sites/stats/site.jsx

```diff
@@ -276,13 +305,15 @@ export default connect(
 	state => {
 		const siteId = getSelectedSiteId( state );
 		const isJetpack = isJetpackSite( state, siteId );
-
+		const isStatsModuleActive = isJetpackModuleActive( state, siteId, 'stats' );
```

Comment:
Cool! Didn't think of this, which we should also check in https://github.com/Automattic/wp-calypso/pull/38975 for the new stats card in the Customer Home.

Reply:
Nice catch for the customer home card, definitely!